Description: Fix spelling errors.
 allows to <verb> -> allows <verb>ing
Author: Bas Couwenberg <sebastic@debian.org>
Forwarded: not-needed

--- a/gui/wxpython/gmodeler/g.gui.gmodeler.html
+++ b/gui/wxpython/gmodeler/g.gui.gmodeler.html
@@ -82,7 +82,7 @@ color="red">16</font>) Show manual and l
 There is also lower menu bar in Graphical modeler dialog where one can
 manage model items, see commands, add or manage model variables,
 define default values and descriptions. Python editor dialog window
-allows to see performation written in Python code. Rightmost tab of
+allows seeing performation written in Python code. Rightmost tab of
 bottom menu is automatically triggered when model is activated and
 shows all the steps of running GRASS modeler modules. In case of some
 errors in calculation process, it is written at that place.
